### ISSUE TITLE
Allow SIP to work over TCP

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7584]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#293|
|Type|enhancement|
|Created|2003-09-29 02:37:44Z|

<details><summary>Searchable metadata</summary>

```
trac-id__293 293
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1064803064000000 1064803064000000
changetime__1467652971867950 1467652971867950
version__ 
owner__dash dash
cc__exarkun cc__itamarst cc__dash
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

```
#!html
<pre>
SIP is pretty much out of my hands.

I don't particularly care about this feature anymore.  Feel free to close it if
you think it will not be immediately useful to someone.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="dash's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|dash commented|
|-|-|

```
#!html
<pre>
BALEETED!
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub commented|
|-|-|

In changeset a36b8f3bbd970b6cf3726be3807672e09eb0db59
```
#!CommitTicketReference repository="" revision="a36b8f3bbd970b6cf3726be3807672e09eb0db59"
Merge pull request #293 from twisted/8560-update-docs-extras

Author: derwolfe
Reviewer: rodrigc
Fixes: #8560

Update documentation for optional extras
```
